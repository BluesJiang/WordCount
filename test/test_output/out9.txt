test\test_cases\test1.in, 字符数: 383
test\test_cases\test1.in, 单词数: 29
test\test_cases\test1.in, 行数: 20
test\test_cases\test1.in, 代码行/空行/注释行: 10/3/9
test\test_cases\test2.in, 字符数: 168
test\test_cases\test2.in, 单词数: 21
test\test_cases\test2.in, 行数: 9
test\test_cases\test2.in, 代码行/空行/注释行: 6/3/0
test\test_cases\test3.in, 字符数: 187
test\test_cases\test3.in, 单词数: 16
test\test_cases\test3.in, 行数: 4
test\test_cases\test3.in, 代码行/空行/注释行: 4/0/0
test\test_cases\test4.in, 字符数: 168
test\test_cases\test4.in, 单词数: 21
test\test_cases\test4.in, 行数: 9
test\test_cases\test4.in, 代码行/空行/注释行: 6/3/0
