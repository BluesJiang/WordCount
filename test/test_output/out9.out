test\test_cases\test1.in, 字符数: 383
test\test_cases\test1.in, 单词数: 29
test\test_cases\test1.in, 行数: 20
test\test_cases\test1.in, 代码行/空行/注释行: 10/3/9
test\test_cases\test2.in, 字符数: 168
test\test_cases\test2.in, 单词数: 21
test\test_cases\test2.in, 行数: 9
test\test_cases\test2.in, 代码行/空行/注释行: 6/3/0
test\test_cases\test3.in, 字符数: 187
test\test_cases\test3.in, 单词数: 16
test\test_cases\test3.in, 行数: 4
test\test_cases\test3.in, 代码行/空行/注释行: 4/0/0
test\test_cases\test4.in, 字符数: 168
test\test_cases\test4.in, 单词数: 21
test\test_cases\test4.in, 行数: 9
test\test_cases\test4.in, 代码行/空行/注释行: 6/3/0
